src/components/Bookings/Bookings.jsx

```diff
@@ -1,16 +1,21 @@
+import React, { useState } from 'react';
 import Search from "@/components/Search/Search";
-import FakeBookings from "@/data/fakeBookings.json";
+import FakeBookings from "@/data/fakeBookings.json"; 
 import SearchResults from "../SearchResults/SearchResults";
```

Comment:
can you be consistent of using your import, in all lines you used "/@/data" in the line 4 you used "../search" ?
you should make line 4 to be similar to line 3 or 2